src/Views/Results/Index.cshtml

```diff
@@ -10,6 +10,14 @@
     + (string.IsNullOrWhiteSpace(Model.FilterModel.query) ? "" : $" - {Model.FilterModel.query}");
 }
 
+@section bodyEnd {
+  @if (Model.FilterModel.DisplayAsMap)
+  {
+    <script src="https://developers.google.com/maps/documentation/javascript/examples/markerclusterer/markerclusterer.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=@(Environment.GetEnvironmentVariable("google_cloud_platform_key"))&callback=initGoogleMaps" async defer></script>
```

Comment:
I think the app should load & validate all environment variables it uses at startup, it should also support non env methods of configuring by using the configuration classes from dotnet.